lib/Logging.js

```diff
@@ -38,24 +39,22 @@ class Logging {
             }));
         }
 
-        if(config.files === undefined) {
```

Comment:
Is this actually a worthwhile change? Was the intention to log no matter what?

Reply:
[Not in PR]